workflow/rocoto/tasks.py

```diff
@@ -10,7 +10,7 @@
 
 
 class Tasks:
-    SERVICE_TASKS = ['arch', 'earc', 'stage_ic', 'cleanup']
+    SERVICE_TASKS = ['arch', 'earc', 'fetch', 'cleanup']
```

Comment:
`stage_ic` should stay here as well:

```suggestion
    SERVICE_TASKS = ['arch', 'earc', 'stage_ic', 'fetch', 'cleanup']
```

Reply:
Made that change